libpod/runtime_img.go

```diff
@@ -276,24 +277,51 @@ func DownloadFromFile(reader *os.File) (string, error) {
 
 // LoadImage loads a container image into local storage
 func (r *Runtime) LoadImage(ctx context.Context, inputFile string, writer io.Writer, signaturePolicy string) (string, error) {
-	if newImages, err := r.LoadAllImageFromArchive(ctx, writer, inputFile, signaturePolicy); err == nil {
+	newImages, err := r.LoadAllImageFromArchive(ctx, writer, inputFile, signaturePolicy)
+	if err == nil {
 		return newImages, nil
 	}
+	logrus.Warnf("Error trying pulling multi-arch images: %v", err)
```

Comment:
- s/multi-arch images/multi-image archives/
- For some formats like `dir`, `LoadAllImageFromArchive` will not even try, so it’s grating to output a warning and then succeed.
- This is the _only_ caller of `LoadAllImageFromArchive` as well as `LoadImageFromSingleImageArchive`, so it seems that there should only be one set of attempts, one list of errors (and the two archive formats should only be attempted once, not once via the multi-image reader and once via the non-reader path).